app/services/current-session.js

```diff
@@ -1,47 +1,42 @@
-/* eslint-disable ember/require-computed-property-dependencies */
 import Service from '@ember/service';
 import { inject as service } from '@ember/service';
-import { get, computed } from '@ember/object';
-import { task, waitForProperty } from 'ember-concurrency';
+import { tracked } from '@glimmer/tracking';
+
+export default class CurrentSessionService extends Service {
+  @service session;
+  @service store;
+
+  @tracked account;
+  @tracked user;
+  @tracked group;
+  @tracked roles = [];
+
+  get canPublish() {
+    return this.hasRole('GelinktNotuleren-publiceerder');
+  }
+
+  get canSign() {
+    return this.hasRole('GelinktNotuleren-ondertekenaar');
+  }
 
-export default Service.extend({
-  session: service('session'),
-  store: service('store'),
   async load() {
     if (this.session.isAuthenticated) {
```

Comment:
Should I prevent this from running more than once and show a warning if it does?

Reply:
I guess that could be a nice addition, but definitely not a requirement

Reply:
Yea it's probably overkill since that will be noticed during PR's anyway.